core-bundle/src/Resources/contao/controllers/FrontendCron.php

```diff
@@ -26,82 +31,13 @@ class FrontendCron extends Frontend
 	 */
 	public function run()
 	{
-		$objResponse = new Response('', 204);
-
-		// Do not run if there is POST data or the last execution was less than a minute ago
-		if (!empty($_POST) || $this->hasToWait())
-		{
-			return $objResponse;
-		}
-
-		$arrLock = array();
-		$arrIntervals = array('monthly', 'weekly', 'daily', 'hourly', 'minutely');
-
-		// Store the current timestamps
-		$arrCurrent = array
-		(
-			'monthly'  => date('Ym'),
-			'weekly'   => date('YW'),
-			'daily'    => date('Ymd'),
-			'hourly'   => date('YmdH'),
-			'minutely' => date('YmdHi')
-		);
-
-		// Get the timestamps from tl_cron
-		$objLock = $this->Database->query("SELECT * FROM tl_cron WHERE name !='lastrun'");
-
-		while ($objLock->next())
+		// Do not run if there is POST data
+		if (empty($_POST))
```

Comment:
To be consistent with the `_cron` route and we're now importing the container anyway, can we use the current request and check if it's a `GET` request maybe?

Reply:
We could maybe even forward to the `/contao/_cron` route but that might be a bit too much just for BC.

Reply:
This check is weaker than the check in the `FrontendController` class, where we explicitly require the method to be `GET`:

```php
$request->isMethod(Request::METHOD_GET)
```

I think we should do the same here – unless we decide to forward as suggested by @Toflar.

Reply:
You're aware this method is not used anymore at all?

Reply:
Yeah, this class is not used at all anymore, that's why I did not put any work into it.

Reply:
Agreed. 👍